src/openforms/validations/tests/test_api.py

```diff
@@ -40,22 +58,44 @@ def setUp(self):
     def test_auth_required(self):
         self.client.logout()
         url = reverse("api:validators-list")
-        response = self.client.get(url)
+        component = "textfield"
+        response = self.client.get(url, args=component)
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
     def test_validations_list(self):
```

Comment:
I think it might be better to have this test check that if no querystring is used, then all validators are returned, then in a separate test check the filters for a specific component.